sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskService.java

```diff
@@ -187,30 +205,40 @@ public Predicate createUserFilter(TaskQueryContext taskQueryContext, TaskCriteri
 		Predicate filter = cb.equal(taskPath.get(Task.CREATOR_USER), currentUser);
 		filter = cb.or(filter, cb.equal(taskPath.get(Task.ASSIGNEE_USER), currentUser));
 
-		Predicate caseFilter = caseService.createUserFilter(
-			new CaseQueryContext(cb, cq, joins.getCaseJoins()),
-			taskCriteria != null
-				? new CaseUserFilterCriteria().excludeLimitedSyncRestrictions(taskCriteria.isExcludeLimitedSyncRestrictions())
-				: null);
+		Predicate caseFilter = isContextCriteriaOrHasNoContext(taskCriteria, TaskContext.CASE)
```

Comment:
this method is always false when is called from either Case, Contact, Event, or TravelEntry. taskCriteria.taskContext is never populated. Due to this situation, some of the tasks are not shown.